vllm/worker/model_runner_base.py

```diff
@@ -202,6 +202,17 @@ def execute_model(
         """
         raise NotImplementedError
 
+    @property
+    def not_compiled_model(self):
+        """
+        Return the model that is not compiled.
+        """
+        if hasattr(self, "_not_compiled_model"):
+            return self._not_compiled_model
+        if hasattr(self, "model"):
+            return self.model
```

Comment:
I think this is error prone since `self.not_compiled_model` can return a compiled model.